containers/selectfriends.js

```diff
@@ -14,36 +14,68 @@ import {
   TouchableHighlight
 } from 'react-native';
 
-import Row from './row.js';
+import Row from '../components/row.js';
 
 const onButtonPress = () => {
-  // do something;
+  // submit if this.state.actual = 0;
+  // send video, send array friends
+  // go to my submissions view
 };
 
-class SelectFriends extends React.Component {
-  constructor(props) {
-    super(props);
 
-    const ds = new ListView.DataSource({rowHasChanged: (r1, r2) => r1 !== r2});
-    this.state = {
-      actual: 3,
-      dataSource: ds.cloneWithRows([
-      {
+class SelectFriends extends React.Component {
+  friendsList = [
+    {
```

Comment:
This mock is not close either to the api response message. In this case you can't ask facebook to change it 🤓